.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Configure '…'
+2. Run '…'
+3. …
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots or command output**
+If applicable, add screenshots or `output logs` to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. Linux]
+ - Python version [e.g. 3.7.5]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/harismuneer/Ultimate-Facebook-Scraper/pullRequest?prid=4591623)

Reply:
This is a Github template